pkg/volumes/do/volumes.go

```diff
@@ -127,6 +140,65 @@ func (a *DOVolumes) FindVolumes() ([]*volumes.Volume, error) {
 	return a.findVolumes(true)
 }
 
+func (a *DOVolumes) findAllVolumes(filterByRegion bool) ([]*volumes.Volume, error) {
+	doVolumes, err := getAllVolumesByRegion(a.DigiCloud, a.region, filterByRegion)
+	if err != nil {
+		return nil, fmt.Errorf("failed to list volumes: %s", err)
+	}
+
+	var myvolumes []*volumes.Volume
+	for _, doVolume := range doVolumes {
+
+		glog.V(2).Infof("Iterating DO Volume with name=%s ID=%s, nametag=%s", doVolume.Name, doVolume.ID, a.nameTag)
+
+		// make sure dropletTags match the volumeTags for the keys mentioned in matchTags
+		tagFound := a.matchDropletTags(&doVolume)
+
+		if tagFound {
+
+			glog.V(2).Infof("Tag Matched for droplet name=%s and volume name=%s", a.dropletName, doVolume.Name)
+
+			var clusterKey string
+			if strings.Contains(doVolume.Name, "etcd-main") {
+				clusterKey = "main"
+			} else if strings.Contains(doVolume.Name, "etcd-events") {
+				clusterKey = "events"
+			} else {
+				glog.V(2).Infof("could not determine etcd cluster type for volume: %s", doVolume.Name)
```

Comment:
I think Infof should be Ok. If at all we have any other tags associated with this volume, the same will just be ignored, rather than showing up as a warn message.